src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE_SEPERATOR = "\t____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("\tHello from\n" + LOGO);
+        System.out.println(LINE_SEPERATOR +
+                "\n\tHello! I'm Duke\n" +
+                "\tWhat can I do for you?\n" +
+                LINE_SEPERATOR);
+        Scanner sc = new Scanner(System.in);
+        TaskList taskList = new TaskList();
+        String userInput = sc.nextLine();
+        while (!userInput.equals("bye")) {
+            try {
+                if (userInput.equals("list")) {
```

Comment:
if-else statements follow coding standards closely. However, you can consider using case statements to implement this huge if-else block